vrxperience_bridge/src/send_dds_done_reply.cpp

```diff
@@ -12,21 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <dds/core/BuiltinTopicTypes.hpp>
 #include "vrxperience_bridge/sim_data_sender.hpp"
 #include "std_msgs/msg/byte_multi_array.hpp"
+#include "DDS_Octets.h"
+
+#define UNUSED(x) (void)(x)
 
 using vrxperience_bridge::SimDataSender;
-typedef SimDataSender<std_msgs::msg::ByteMultiArray, dds::core::BytesTopicType> DDSDoneReplySender;
+typedef SimDataSender<std_msgs::msg::ByteMultiArray, DDS_Octets> DDSDoneReplySender;
 
-void convert(std_msgs::msg::ByteMultiArray IN rosMsg, dds::core::BytesTopicType OUT simMsg)
+void convert(std_msgs::msg::ByteMultiArray IN rosMsg, DDS_Octets OUT simMsg)
 {
-  simMsg.data(rosMsg.data);
+  UNUSED(rosMsg);
+
+  // TODO Copy data before sending out the message
```

Comment:
Copying data has been purposely left out for now. Message payload is not relevant for correct functioning of the bridge, and the implementation will require looking into memory management in Cyclone DDS not to introduce accidental memory leak.

Reply:
Should work fine, according to your understanding. OK with me.